data/json/monsters/mutant_mammal.json

```diff
@@ -151,35 +124,51 @@
   {
     "id": "mon_cat_mutant_kitten_prism",
     "type": "MONSTER",
-    "copy-from": "mon_cat_mutant_prism",
     "name": { "str": "iridescent kitten" },
     "description": "In the light, this adorably tiny bundle of unkempt fluff is an ever-changing flurry of brilliant colors.",
+    "copy-from": "mon_cat_kitten",
     "default_faction": "cat",
-    "bodytype": "dog",
-    "categories": [ "WILDLIFE" ],
-    "species": [ "MAMMAL" ],
-    "volume": "1000 ml",
-    "weight": "1 kg",
-    "hp": 6,
-    "material": [ "flesh" ],
-    "symbol": "c",
-    "color": "light_gray",
     "luminance": 4,
+    "//": "Higher dodge due to the constantly changing fur pattern",
+    "dodge": 15,
     "looks_like": "mon_cat_kitten",
-    "melee_damage": [ { "damage_type": "cut", "amount": 2 } ],
-    "dodge": 14,
-    "special_attacks": [ { "id": "scratch", "attack_upper": false, "cooldown": 10 } ],
-    "fear_triggers": [ "SOUND", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
-    "placate_triggers": [ "MEAT" ],
-    "harvest": "mammal_tiny",
-    "petfood": {
-      "food": [ "CATFOOD" ],
-      "feed": "The %s seems to like you!  Or maybe it just tolerates your presence better.  It's hard to tell with felines.",
-      "pet": "The %s softly purrs as you pet it.  It demands more."
-    },
     "flags": [ "SEES", "HEARS", "GOODHEARING", "SMELLS", "ANIMAL", "CANPLAY", "PATH_AVOID_DANGER_1", "WARM", "HIT_AND_RUN" ],
     "upgrades": { "age_grow": 120, "into": "mon_cat_mutant_prism" }
   },
+  {
+    "id": "mon_cat_liquid_kitten",
+    "type": "MONSTER",
+    "copy-from": "mon_cat_kitten",
+    "name": { "str": "viscous kitten" },
+    "description": "It is a universally accepted fact that cats are liquid but this kittens body seems to quiver ever so slighty while stumbling around and mewing.",
+    "upgrades": { "age_grow": 120, "into": "mon_cat_liquid" }
+  },
+  {
+    "id": "mon_cat_liquid",
+    "type": "MONSTER",
+    "copy-from": "mon_cat",
+    "name": { "str": "liquid cat" },
+    "description": "It is universally known that cats are liquid but this cats body is actually quivering while it walks around.  It also enjoys resting while twisting its body in ways that would break a normal cats bones.",
```

Comment:
```suggestion
    "description": "It is universally known that cats are liquid but this cats body is actually quivering while it walks around.  It also enjoys resting while twisting its body in ways that would break a normal cat's bones.",
```